index.html

```diff
@@ -11,21 +11,55 @@
 </head>
 
 <body>
+  <nav class="nav">
+    <div class="d-flex nav-child">
+      <div><a href="#list">Awesome Books</a></div>
+      <ul class="right-nav d-flex">
+        <li><a href="#list">List</a></li>
+        <li><a href="#new">Add New</a></li>
+        <li><a href="#contact">Contact</a></li>
+      </ul>
+    </div>
+    <div class="date"></div>
```

Comment:
- [OPTIONAL] This is not really optional, but since it is a minor issue, I'll leave it here as a suggestion. This date ought to have been aligned to the right.

Reply:
Thanks bro.